Tasks/Maven/mavenHelper.ps1

```diff
@@ -104,10 +105,15 @@ function RunSonarQubeAnalysis
 			# The platform may cache the db details values so we force them to be empty
 			$sqArguments = CreateSonarQubeArgs $sqServiceEndpoint.Url $sqServiceEndpoint.Authorization.Parameters.UserName $sqServiceEndpoint.Authorization.Parameters.Password "" "" ""
 		}
-
+		
 		$sqArguments = $userOptions + " " + $sqArguments
 		Write-Verbose "Running Maven with goal sonar:sonar and options: $sqArguments"
-
+		
+		if($execFileJacoco)
+		{
+			$sqArguments = $sqArguments + " -Dsonar.jacoco.reportPath=" + $execFileJacoco			
```

Comment:
Have you tested this when $execFileJacoco contains spaces or special characters?  I wonder if the path needs to be wrapped in quotes/apostrophes.